create_new_module_helper.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Prompt user for project name
+read -p "Enter new module name: " module_name
+
+# Prompt user for application id
+read -p "What kind of module you would like to create: [1] compose [2] desktop [3] KMM: " option
+
+case $option in
+1)
+  plugin_name=compose-module-setup
+  ;;
+2)
+  plugin_name=desktop-application-setup
+  ;;
```

Comment:
I think this one is not necessary either.
If I'm not mistaken the desktop setup should be done only one time in the application module.
All the subsequent presentation modules can be created using the `compose-module-setup`